.gitignore

```diff
@@ -25,5 +31,4 @@ captures/
 lint/intermediates/
 lint/generated/
 lint/outputs/
-lint/tmp/
-
+lint/tmp/
```

Comment:
Probably it's redundant as well, since it's a "top-level" `.gitignore` file. Lint files are generated under `<module>/build/reports` and it's currently excluded by `app/.gitignore` file, note `/build` inside.

Reply:
Seems like you're right, so I will remove it.

Reply:
Done